lib/actions/serial.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+/**
+ * @module lib/actions/serial
+ * @author Eric Satterwhite
+ **/
+
+const assert = require('assert')
+const {typeOf, object} = require('@logdna/stdlib')
+
+const NAME = 'SetupChain.serial'
+const TIMES_ERR = `${NAME} requires 'times' to be an integer`
+const ACTION_ERR = `${NAME} 'action' invalid`
+
+module.exports = async function serial(times, action, opts) {
+  assert.equal(typeOf(times), 'number', TIMES_ERR)
```

Comment:
I wonder if we should instead use `Number.isFinite()` over the type check, to avoid `Infinity` and `NaN`. maybe enforce `> 0` too.